profiles/disa_stig-rhel7-baseline/controls/V-71859.rb

```diff
@@ -110,8 +110,9 @@
   [org/gnome/login-screen]
   banner-message-enable=true"
 
-  describe command("grep banner-message-enable /etc/dconf/db/local.d/*") do
-    its('stdout') { should cmp 'banner-message-enable=true'}
+  only_if { command('dconf').exist? }
+
+  describe command("dconf read /org/gnome/login-screen/banner-mesage-enable") do
```

Comment:
@jeannegreulich This *should* be OK since the `locks` are supposed to override everything and that's what we're actually checking. If this comes back incorrectly, then the setting simply is not correct since the user can override it.